### ISSUE TITLE
Print definitions in scope in robot context

### ISSUE THREAD

**Issue by xsebek** [swarm-game/swarm#1129]
From #7:
> It may be necessary to add something to keep track of the order of the definitions (something the `robotEnv` does not care about) so that we can pretty-print them to a file in the same order the user entered them.

Add a panel showing the robot context of the focused robot to UI.

Ideally, use it in the `robot` Web API endpoint so we can inspect the robot context at runtime.

**Comment by byorgey** [comment]
I think some of my previous comments were confused about this, but note that an `Env` is *not* the right thing to use to recover the *definitions* of terms.  An `Env` tracks runtime *values* of variables.  Contributing to the confusion is the fact that in the case of variables denoting functions defined via `def`, we can sometimes do an OK job extracting a definition from its runtime value, because its value is simply a delayed expression.  However, even in that case its value has been elaborated to contain things like `force` and so on.  What we really need is simply an ordered map from names to definitions.

**Comment by xsebek** [comment]
I think the idea was to get the current available definitions of the base robot.

For other robots it doesn't matter since they cannot be resumed with their context.

If I can find time to get back to outputting REPL replay, that should be an alternative solution to the env store idea.

**Comment by xsebek** [comment]
Yeah, I imagine it might be useful to see the pretty-printed values in executing robot context.
